beacon_node/src/config.rs

```diff
@@ -887,6 +887,14 @@ pub fn get_config<E: EthSpec>(
         .max_gossip_aggregate_batch_size =
         clap_utils::parse_required(cli_args, "beacon-processor-aggregate-batch-size")?;
 
+    if let Some(delay) = clap_utils::parse_optional(cli_args, "delay-block-publishing")? {
+        client_config.chain.block_publishing_delay = Some(Duration::from_secs(delay));
```

Comment:
I think we could use `from_secs_f64` here to allow us to play with seconds in floating points?

Reply:
Done in https://github.com/sigp/lighthouse/pull/6947/commits/20368f0fc84c5bd07bc89e94cd7ee6cccd61f78f